workers/unity/Packages/com.improbable.gdk.mobile/Utility/LaunchArguments.cs

```diff
@@ -30,7 +32,25 @@ public static class LaunchArguments
                 UnityEngine.Debug.LogException(e);
             }
 
-            return new Dictionary<string, string>();
+            return null;
+        }
+
+        public static Dictionary<string, string> GetiOSArguments()
+        {
+            try
+            {
+                var argumentsEnvVar = System.Environment.GetEnvironmentVariable(LaunchArgumentsEnvKey);
```

Comment:
do the same change that we did for Android: checking whether we are in the editor as that means we can just return immediately. In this case, it should be fine as we are not calling any java objects, but just to be consistent

Reply:
👍